pkg/controller/unifiedpushserver/unifiedpushserver_controller.go

```diff
@@ -100,51 +103,184 @@ func (r *ReconcileUnifiedPushServer) Reconcile(request reconcile.Request) (recon
 		return reconcile.Result{}, err
 	}
 
-	// Define a new Pod object
-	pod := newPodForCR(instance)
+	secret := newPostgresSecret(instance)
```

Comment:
This stuff all needs to be cleaned up, it was just getting something working (mostly)